TTS/tts/models/xtts.py

```diff
@@ -436,11 +482,6 @@ def synthesize(self, text, config, speaker_wav, language, **kwargs):
             as latents used at inference.
 
         """
-
-        # Make the synthesizer happy 🥳
-        if isinstance(speaker_wav, list):
-            speaker_wav = speaker_wav[0]
```

Comment:
This looks like a breaking API change?

I foresee people upgrading their libraries without realizing `speaker_wav` (singular!) has changed to always be a list, and since strings can be iterated over, they'll get cryptic errors...

Reply:
Currently, it supports both string path or list: https://github.com/coqui-ai/TTS/pull/3149/files#diff-9e55c2563c46d07b849d20125c940929544d253136bea5835e83544d291e12e0R434-R437